packages/@uppy/companion/tsconfig.json

```diff
@@ -7,7 +7,9 @@
     "sourceMap": false,
     "allowJs": true,
     "checkJs": true,
-    "noEmitOnError": true
+    "noEmitOnError": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true
```

Comment:
I suspect this could be the issue to failing Travis tests, since Companion’s `lib` folder is now built weirdly locally for me.

Do we have any tests that test the `lib` folder of Companion, how it’s used when installed from `npm`?

Reply:
I have removed them as they are not needed in this PR anyways